### ISSUE TITLE
Support for hostnames instead of just IP addresses

### ISSUE THREAD

**Issue by wlabarron** [ents-crew/x32-reflector#7]
This may introduce too much overhead into the program or be out of scope, but thought I’d bring it up anyway.

By allowing you to enter a host name instead of an IP address, you reduce the chance of the connection being broken if a DHCP lease expires for some reason (perhaps rebooting faulty equipment mid-event).

**Comment by Vitineth** [comment]
This should be fine to implement - just need to remove the IP regex and it should work exactly the same as it uses https://nodejs.org/api/dgram.html#socketsendmsg-offset-length-port-address-callback internally which should resolve any hostnames automatically and the system will handle any caching etc.

**Comment by Vitineth** [comment]
Blocked by #6
